test/rules-ceddl-fullstory.spec.ts

```diff
@@ -86,12 +85,6 @@ describe('Ruleset: CEDDL to FullStory', () => {
         expectUndefined(payload, 'linkedProduct');
       });
 
-      it('does not send CEDDL cart item products to FS.event on load', async () => {
```

Comment:
This test didn't fail after removing `readOnLoad: false` from the CEDDL ruleset. It turns out this test stopped verifying the target behavior when I hybridized ruleset tests to run in a Node env or via Playwright browser automation. For this test to have been verifying the target behavior, we would've needed to specify a global `readOnLoad: true` configuration to be overridden by the rule's `readOnLoad: false`.

Either way, I think it makes sense to remove this test now since there's no longer an exceptional behavior to verify.